src/monda/tests/sorad_retrieval_qc_plot_test.py

```diff
@@ -118,91 +126,73 @@ def run_example(platform_id = 'PML_SR004',
                                   timewindow = (datetime_i, datetime_e),
                                   layer=layer, bbox=bbox)
 
-
         log.info(f"{response['length']} features received.")
 
         if response['length'] == 0:
             continue
-
+    
         file_id = f"{datetime_i.strftime('%Y-%m-%d')}_{rrsalgorithm.upper()}"   # labelling for output data files and plots
 
         rrswl, time, lat, lon, rel_view_az,\
                ed, ls, lt, rrs,\
                sample_uuids, platform_ids, platform_uuids,\
-               gps_speeds, tilt_avgs, tilt_stds = unpack_response(response, rrsalgorithm, wl_output)
+               gps_speeds, tilt_avgs, tilt_stds = access.unpack_response(response, rrsalgorithm, wl_output)
 
         if output_plots:
             log.info("Creating (ir)radiance plots")
             plots.plot_ed_ls_lt(ed, ls, lt, time, wl_output, file_id, target)
 
-
+        # Step (0) QC filters based on relative aziumuth and tilt/tilt std
+        q_az =  qc.rel_az_filter(rel_view_az, lower_azi_bound = 110, upper_azi_bound = 170)
```

Comment:
suggest to use 90-150 as these are likely to give better results than including 170